src/Microsoft.ComponentDetection.Detectors/pip/SimplePythonResolver.cs

```diff
@@ -292,7 +300,7 @@ private async Task<IList<PipGraphNode>> ProcessQueueAsync(ISingleFileComponentRe
     private async Task<IList<PipDependencySpecification>> FetchDependenciesFromPackageStreamAsync(string name, string version, Stream packageStream)
     {
         var dependencies = new List<PipDependencySpecification>();
-        using var package = new ZipArchive(packageStream);
```

Comment:
We should probably update this code to not pass around streams which can be disposed.

Reply:
should we create a separate issue to work on this? Seems it might be a bigger change. We would need to update the in resolver, client, cache, and tests

Reply:
Yes. I'm not sure of the entirely familiar with how these streams are used, but I believe if they're always parsed as `ZipArchive`, then we should just be storing `ZipArchive`s and not the underlying `MemoryStream`.